plugins/out_s3/s3.c

```diff
@@ -120,8 +119,9 @@ static char *mock_error_response(char *error_env_var)
     return NULL;
 }
 
-/* static void s3_retry_warn (struct flb_s3 *ctx, struct s3_file *chunk, char create_time[80], struct multipart_upload *m_upload){
-    strftime(create_time, 80, "%Y-%m-%d %H:%M:%S", localtime(&(chunk->create_time)));
+static void s3_retry_warn (struct flb_s3 *ctx, struct s3_file *chunk, char create_time[20], struct multipart_upload *m_upload){
```

Comment:
why do you need to pass create time into the function? Could this just be a local variable

Reply:
I think create_time should just be a variable that is in s3_retry_warn function